cmake/Dependencies.cmake

```diff
@@ -92,12 +92,39 @@ endif()
 
 # ---[ Python
 if(BUILD_python)
-  # disable Python 3 search
-  find_package(PythonInterp 2.7)
-  find_package(PythonLibs 2.7)
-  find_package(NumPy 1.7.1)
-  find_package(Boost 1.46 COMPONENTS python)
-
+  if("${python_version}" VERSION_GREATER "2.99")
```

Comment:
How about `GREATER` `OR` `EQUAL`, or `NOT` `LESS` (as in http://stackoverflow.com/questions/16667017/cmake-express-the-greater-or-equal-statement)?